web/src/routing/routes/ForgotPassword/ForgotPasswordView/ForgotPasswordForm/styles.ts

```diff
@@ -0,0 +1,11 @@
+import { makeStyles } from "@mui/styles";
+
+export const useStyles = makeStyles({
+  buttonProgress: {
+    position: "absolute",
+    top: "50%",
+    left: "50%",
+    marginTop: -10,
```

Comment:
From my experience negative margin values sooner or later makes CSS hard to maintain. Can we refactor it somehow?

Reply:
Honestly, I was following the styles pattern from the other forms, it causes something like that: 
<img width="449" alt="image" src="https://user-images.githubusercontent.com/25722645/169235377-6e3c3761-631c-41b4-b8a0-3c6280637bd6.png">

I wonder if it should be like that. I can refactor code to showing only spinner instead whole form when loading. Or maybe loading button. Then these styles are no longer needed.

I can't see loading state on designs, so it's hard to manage.

Reply:
I'd suggest to set spinner in button then, so it will be visible instead of "SIGN IN" text